src/loader.js

```diff
@@ -16,6 +16,14 @@ import schema from './loader-options.json';
 const pluginName = 'mini-css-extract-plugin';
 
 function hotLoader(content, context) {
+  const renderImport = (name, modulePath) => {
+    const request = loaderUtils.stringifyRequest(context.context, modulePath);
+
+    if (context.esModule) {
+      return `import ${name} from ${request};`;
```

Comment:
@alexander-akait  not sure if webpack module interop will work here, mb need to change to `* as $name`